src/System.Xml.ReaderWriter/tests/XmlReader/Tests/AsyncReaderLateInitTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Xml;
+using System.IO;
+using Xunit;
+using Xunit.Extensions;
+
+namespace XMLTests.ReaderWriter
+{
+    public static class AsyncReaderLateInitTests
+    {
+        private const string _dummyXml = @"<?xml version=""1.0""?>
+                <root>
+                    <a/><!-- comment -->
+                    <b>bbb</b>
+                    <c>
+                        <d>ddd</d>
+                    </c>
+                </root>";
+
+        private static Stream GetDummyXmlStream()
+        {
+            byte[] buffer = Encoding.UTF8.GetBytes(_dummyXml);
+            MemoryStream ms = new MemoryStream(buffer);
+            if (ms.Position != 0)
+                throw new NotImplementedException();
```

Comment:
That is leftover from development where I wasn't sure if Position is 0 and just threw some random exception. Thanks for finding it!